templates/plugin/gutterball/foreman_gutterball_url.yaml.erb

```diff
@@ -0,0 +1,6 @@
+:foreman_gutterball_url:
+  :protocol: <%= @protocol %>
```

Comment:
Does the plugin use these variables? I'd only parameterize the configuration parameters that the plugin actually uses.